apps/web/.babelrc

```diff
@@ -11,7 +11,6 @@
   ],
   "plugins": [
     "@emotion",
-    "babel-plugin-styled-components",
```

Comment:
removed the `styled-components` and all related tooling in favor to `@emotion/styled`

Reply:
Fantastic!